docs/area-coordinators.md

```diff
@@ -48,7 +48,7 @@ The current Area Coordinators are:
 ### Upcoming Meetings
 
 - 2019-12-04 - Production Support (Ken Herner)
-- 2019-12-11 - Software (Brian Lin)
+- 2019-12-11 - [Software](https://docs.google.com/document/d/125dJS8BPYOJKHAi5Bmn0I0Z4X9nGe2Wjuq09Z9zY0I8/edit?usp=sharing) (Brian Lin)
```

Comment:
This is a nitpick, so you may merge without change, but I prefer to omit the needless crap after the document ID (e.g., `edit?usp/sharing`).

Reply:
FWIW, I've had experiences in the past where messing with the share URL has somehow broken privs for people so I always include it as it's given

Reply:
FWIW, I've had experiences in the past where messing with the share URL has somehow broken privs for people so I always include it as it's given